client/config.md

```diff
@@ -115,6 +117,34 @@ transport:
     secret: SS_SECRET
 ```
 
+Note that Websockets is not yet supported on Windows. In order to have a single config for all platforms, use a first-supported endpoint:
+
+```yaml
+transport:
+  $type: tcpudp
+  tcp:
+    $type: shadowsocks
+    endpoint:
+      $type: first-supported
```

Comment:
What about `first-success`?

Reply:
It's a bit tricky. It requires testing each of them. Then we may need to update the list too. We can add that later, since it's a can of worms.

Reply:
Sounds good.